src/utils/packet.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const IlpPacket = require('ilp-packet')
+const assert = require('assert')
+const { omitUndefined } = require('../utils')
+const base64url = require('../utils/base64url')
+const debug = require('debug')('ilp:packet')
+
+const serialize = IlpPacket.serializeIlpPayment
+const parse = (packet) => {
+  try {
+    return IlpPacket.deserializeIlpPayment(packet)
+  } catch (e) {
+    debug('error while parsing packet: ' + e.message)
+    return undefined
+  }
+}
+  
+function getFromTransfer (transfer) {
+  assert(transfer, 'transfer must be defined. got: ' + transfer)
+  assert(typeof transfer === 'object', 'got invalid transfer: ' + transfer)
+  assert(Buffer.isBuffer(transfer.ilp || transfer.data),
```

Comment:
the `ilp-packet` module returns a buffer, and parses buffers. Shouldn't the LPI use them too?